### PR TITLE
Missing topics 2.1

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description

1. Alerts & Dialogs
2. PDFviewer
3. Bottom Sheet
4. Audio widgets (player, record audio, play stop sound)
5. Biometric action

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-573 

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Enhancement to current docs
- [ ] Removed outdated references
- [ ] Update assets

### CODE REVIEW

File: docs/ff-concepts/alerts-notification/alert-dialog.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+</div>
+<p></p>
+
+### Types of Alert Dialog
+
+We allow you to define two types of Alert Dialog Actions:
+
+- **Informational Dialog:** To show some information the user should be aware of before interacting with the app. Contains only a single action button.
+- **Confirm Dialog:** This dialog can contain two action buttons. It can trigger the subsequent action based on whether a user confirms the action. It can also be used before performing any non-revertable user action, for example, before deleting a user account.
+- **Custom Dialog**: This is a fully customizable dialog that you can create using [components](../../ui/components/overview.md).
```
Comment (PoojaB26) [inline docs/ff-concepts/alerts-notification/alert-dialog.md:45]: Link doesnt work.

File: docs/ff-concepts/alerts-notification/alert-dialog.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+---
```
Comment (PoojaB26) [inline docs/ff-concepts/alerts-notification/alert-dialog.md:1]: Can you check the component URLs, they dont link to anything.

File: docs/ff-concepts/file-handling/audio/audio-player.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+3. Find the **Title** property and enter the value to display the audio title. You can also set this from the variable, such as the app state variable, API response variable, Firestore document, and more. To do so, click on the **Set from Variable**.
+4. Find the **Path** property and enter the **URL** for the new audio file.
+5. By default, the audio stops when you move to another page using the **Pause on Forward Navigation** property.
+6. When an app goes in the background while the audio player is running, you can control the audio player's behavior using the **Play in Background** property, which has following options:
+    1. **Enabled**: This keeps the audio player running.
+    2. **Disabled, restore on foreground**: This pauses the audio player and starts playing again as soon as the app is visible.
+    3. **Disabled, pause**: This stops the audio player as soon as the app goes in the background.
+
+:::info
+By default, every AudioPlayer widget added to the project will show sample audio from the internet.
```
Comment (PoojaB26) [inline docs/ff-concepts/file-handling/audio/audio-player.md:28]:
```suggestion
By default, every AudioPlayer widget added to the project will play a sample audio file from a network URL.
```

File: docs/ff-concepts/navigation-routing/bottom-sheet.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+Below are the types of Bottom Sheet actions:
+
+1. **Show**: This opens the bottom sheet.
+2. **Dismiss**: This closes the bottom sheet.
+
+## Opening Bottom Sheet
+
+Follow the steps below to add an action that opens the bottom sheet:
+
+1. Create a bottom sheet [component](../../resources/ui/components/overview.md). Tip: you can also create one from the 'BottomSheet' templates.
```
Comment (PoojaB26) [inline docs/ff-concepts/navigation-routing/bottom-sheet.md:50]: Create this tip into an admonition. And then also link "create components from templates" section under components.

**Review by PoojaB26 — APPROVED** [review]
Nit changes. Please fix them and then you can merge.

**Comment by pinkeshmars** [comment]
All review comments addressed!

**Pull request merged**